olive/auto_optimizer/config_template/bert.yaml

```diff
@@ -0,0 +1,16 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We should not need config templates to determine a config parameters of a Pass.

Reply:
These kind of model.yaml and tempalte_mapping.py are only for opt_level 0 where we collect the verified passes sequences(flows) and configs to speed up optimization.

Reply:
And the feature of auto optimization will be iterated actively with more generic strategies.
![image](https://github.com/microsoft/Olive/assets/13343117/1d637f70-c654-45fe-a3ca-73f4964fde1b)